protocol_translation.jl

```diff
@@ -9,74 +9,6 @@ license_text = "# This file is a part of GraknClient.  License is MIT: https://g
 
 reg_license = r"( \*.+\n)|(\/\*\n)|( \*\n)"
 
-#empty the source code directory
-rm(root_dir;force = true, recursive = true)
-#make a new source code directory 
-mkdir(root_dir)
-
-#read the conent of the java directory and throw away all not necessary
-dirs_java = readdir(root_dir_java)
-result_dirs_java = filter(x->(!startswith(x,'.') && (!isfile(joinpath(root_dir_java,x)) && (x != "docs"))  ),dirs_java) 
-
-#copy the source structure from java to the julia grakn client
-for dir in result_dirs_java 
-    cp(joinpath(root_dir_java * "/" * dir), root_dir * "/" * dir)
-end
-
-#get the GraknClient.jl and bring it in place
-download("https://raw.githubusercontent.com/Humans-of-Julia/GraknClient.jl/main/src/GraknClient.jl", root_dir * "/GraknClient.jl")
-
-function prepare_java_file(full_filepath::String)
-    #read the given file
-    text_lines = readlines(full_filepath;keep=true)
-
-    #prepare the first lines
-    result_lines = [license_text, "\n"]
-
-    #take the text array, go trough an let beside the 
-    #license lines 
-
-    #put the rest of the lines on top of the prepared 
-    #result_lines array
-    for i in 1:length(text_lines)
-        test = match(reg_license, text_lines[i])
-        if test === nothing
-            tmp_line = "# " * text_lines[i]
-            push!(result_lines, tmp_line)
-        end
-    end
-    #join the text-array to a string
-    text_result = join(result_lines,"","")
-
-    # write back file
-    file_io = open(full_filepath,"w")
-    write(file_io, text_result)
-    close(file_io)
-
-    #change filename
-    mv(full_filepath, replace(full_filepath,".java"=>".jl"))
-end
-
-#do the work to change the files
-for (root, dirs, files) in walkdir(root_dir)
-    for file in files 
-        if !occursin(".git", root) && !occursin(".github", root) && endswith(file, ".java")
-            prepare_java_file(joinpath(root, file)) 
-            println(joinpath(root, file))
-        end
-    end
-end
-
-#remove the BUILD Files
-for (root, dirs, files) in walkdir(root_dir)
-    for file in files 
-        if file == "BUILD"
-            rm(joinpath(root,file))
-            println(joinpath(root,file), " : deleted")
-        end
-    end
-end
-
 #make the generated folder
 mkdir(root_dir * "/generated")
```

Comment:
In the future, this should be using `joinpath`

Reply:
Yes, you are right. I will change this the next time.